api_textual/rest/texts.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import permissions, serializers, viewsets
+
+from .. import models
+
+
+class TextSerializer(serializers.HyperlinkedModelSerializer):
+    createdAt = serializers.DateTimeField(source='created_at')
+
+    class Meta:
+        model = models.Text
+        fields = (
+            'id',
+            'name',
+            'slug',
+            'text',
+            'createdAt',
+        )
+
+
+class TextViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = models.Text.objects.all()
+    serializer_class = TextSerializer
+    permission_classes = [permissions.AllowAny]
```

Comment:
This seems to be part of every view set. No need to do any action here now, just sharing. Basically, we will ever need only three types of permissions:

1. Read only public
2. Read/Write for specific user
3. Read only statistical data for admin